cob_bringup/robots/raw3-1.xml

```diff
@@ -58,7 +59,7 @@
 			<arg name="robot" value="$(arg robot)"/>
 			<arg name="sim" value="$(arg sim)"/>
 		</include>
-		<include file="$(find cob_bringup)/components/legacy_schunk_powercube.launch">
+		<include file="$(find cob_bringup)/components/canopen_generic.launch">
```

Comment:
The problem is that "someone" deleted legacy_schunk_powercube.launch so now the software that we have in the main repository is wrong fro raw3-1 and cob3-6. Anyway I will put this back as it was...